apps/web/components/examples/blockchain/api-client-demo.tsx

```
@@ -0,0 +1,368 @@
+            </button>
+          </div>
+
+          {/* Journey Results */}
+          {journeyResults && (
+            <div className="mt-6 bg-gray-50 rounded-lg p-4">
+              <h3 className="font-medium text-gray-900 mb-3">
+                User Journey Results
+              </h3>
+              
+              <div className="space-y-2">
+                {journeyResults.steps.map((step: any, index: number) => (
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)